package.json

```diff
@@ -2,7 +2,10 @@
   "name": "backend-capstone",
   "version": "1.0.0",
   "main": "src/app.js",
-  "repository": "git@github.com:ReCoded-Org/backend-capstone-template.git",
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/ReCoded-Org/backend-capstone-template.git"
+  },
   "author": "Ammar-64",
   "license": "MIT",
   "scripts": {
```

Comment:
To differentiate between testing and development, and avoid starting a server while testing. This could be done by doing the following: 
replace `"start": "nodemon src/app.js"` with `"start": "export NODE_ENV=dev && nodemon src/app.js",`
and "test": `"yarn validate && jest --passWithNoTests"` with `"test": "export NODE_ENV=test && yarn validate && jest --passWithNoTests"`